app/helpers/formatting_helper.rb

```diff
@@ -27,7 +27,14 @@ def extract_status_plain_text(status)
   module_function :extract_status_plain_text
 
   def status_content_format(status)
-    html_aware_format(status.text, status.local?, preloaded_accounts: [status.account] + (status.respond_to?(:active_mentions) ? status.active_mentions.map(&:account) : []))
+    MastodonOTELTracer.in_span('HtmlAwareFormatter rendering') do |span|
+      span.add_attributes(
+        'app.formatter.content.type' => 'status',
+        'app.formatter.content.origin' => status.local? ? 'local' : 'remote'
+      )
+
+      html_aware_format(status.text, status.local?, preloaded_accounts: [status.account] + (status.respond_to?(:active_mentions) ? status.active_mentions.map(&:account) : []))
```

Comment:
We discussed distinguishing status text renders from account bio renders, because the former always preload mentioned accounts while the latter never do.

I am not aware of another distinction here, but if the performance data we gather with this points us in that direction, we could investigate further I think.